code/modules/mob/living/carbon/life.dm

```diff
@@ -45,12 +52,11 @@
 	if(stat == DEAD)
 		return
 
-	if(health <= CONFIG_GET(number/health_threshold_dead))
+	if(health <= get_death_threshold())
 		death()
 		return
 
-	var/crit_threshold = ishuman(src) ? CONFIG_GET(number/health_threshold_crit) : 0
-	if(knocked_out || sleeping || getOxyLoss() > 50 || health < crit_threshold)
+	if(knocked_out || sleeping || getOxyLoss() > 50 || health < get_crit_threshold())
```

Comment:
whats the 50 for?

Reply:
Some pre-existing feature that makes the mob pass out due to lack of oxygenation I guess.